src/data/utils.js

```diff
@@ -23,6 +23,17 @@ const getAllPossibleQueryParams = (locationURl = null) => {
   );
 };
 
+export const handleURLUpdationOnLoad = pageName => {
+  const queryParam = getAllPossibleQueryParams();
+  if (!Object.prototype.hasOwnProperty.call(queryParam, AUTH_MODE)
+    || queryParam?.[AUTH_MODE] !== pageName) {
+    const url = new URL(window.location.href || process.env.LMS_BASE_URL);
```

Comment:
It should be `getConfig().MARKETING_SITE_BASE_URL` instead of `process.env.LMS_BASE_URL`. `MARKETING_SITE_BASE_URL` value is setting in [edx-internal](https://github.com/edx/edx-internal/blob/master/frontends/common/prod_config.yml#L15C3-L15C26)